tests/test_cmdline.py

```diff
@@ -667,6 +672,59 @@ def test_xml(self):
             cov.xml_report(morfs=["mod1", "mod2", "mod3"])
             """)
 
+    def test_json(self):
+        # coverage json [-i] [--omit DIR,...] [FILE1 FILE2 ...]
+        self.cmd_executes("json", """\
+                    cov = Coverage()
```

Comment:
These should all be indented just one stop, as in the rest of the file.

Reply:
addressed